aten/src/ATen/native/cuda/Reduce.cuh

```diff
@@ -200,9 +200,11 @@ template <typename index_t>
 static OffsetCalculator<2, index_t> make_output_calculator(const TensorIterator& iter) {
   int num_reduce_dims = iter.num_reduce_dims();
   int num_output_dims = iter.ndim() - num_reduce_dims;
+  int input_index = iter.ntensors() - 1;
+  int output_index = 0;
   std::array<const int64_t*, 2> strides = {
-    iter.strides(0).data() + num_reduce_dims,
-    iter.strides(1).data() + num_reduce_dims,
+    iter.strides(output_index).data() + num_reduce_dims,
+    iter.strides(input_index).data() + num_reduce_dims,
```

Comment:
what's going on here?  This isn't ever looking at the strides of the other outputs?

Reply:
Iterator created with outputs that have same shapes. So I just need metrics(like stride) from just one of the output to navigate, the second will have same one.

Reply:
where do you actually check that the strides of the outputs are the same?

Reply:
+1 -- can you respond to this comment? How do we know the strides will be the same? Shape can be the same, and have different strides.

E.g.

```
x=torch.empty(10,5)
y=torch.empty(5,10).t()
# Now x and y have the same shape, but different strides
z=torch.randn(10,5,100)
torch.std_mean(z, result1=x, result2=y)
```

Reply:
It's a part of TensorIterator.cpp (line 518)